core/types.ts

```diff
@@ -0,0 +1,8 @@
+import type { StoreValue } from 'nanostores';
```

Comment:
We use `types.ts` in Nano Stores for [type tests](https://github.com/ai/check-dts/).

Can you move it to `index.d.ts`?

Reply:
This file contains common type models for the "useStore" and "useStoreSignal" hooks

Reply:
Maybe it is better to put everything in single `index.d.ts` and `index.js`. This project is very small to use practices for huge projects.